demo/youtube-demo/google-youtube.js

```diff
@@ -0,0 +1,180 @@
+/*
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+
+/**
+ * This element supports a subset of the features of the google-map element
+ * v2.x. Namely, the fluid attribute and videoId/list properties are supported.
+ */
+class GoogleYoutube extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        :host {
+          display: block;
+        }
+
+        :host([fluid]) {
```

Comment:
Are we using this attribute anywhere? I didn't find it in the demo with a find-in-page but maybe I'm holding it wrong. If we don't need it for the demo, maybe just nuke it to keep it simplerer?